pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -20,8 +20,8 @@ import (
 )
 
 const (
-	// defaultHelperImage is default image of helper
-	defaultHelperImage = "busybox:1.26.2"
+	// DefaultHelperImage is default image of helper
+	DefaultHelperImage = "busybox:1.31.1"
```

Comment:
Is this necessary? I think change this image will rolling-update some clusters

Reply:
ok, I will revert this.

Reply:
not revert yet?

Reply:
done